src/hooks/use-dao-details.ts

```diff
@@ -78,6 +78,7 @@ export function useDAODetails(id: string) {
         symbol: "",
         decimals: 0,
         image_url: "",
+        max_supply: 10000000,
```

Comment:
Why is this hardcoded? And why 10M?

The default will be 6 decimals and 1B supply, but we should check or have a way to look this up if we need it.